### PR TITLE
add the keys to the keywallet type

### PR DESCRIPTION
**Pull request by Geometer1729**
Closes # .

Downstream we want to test our CLI with plutip. The cli reads wallets from files so we need a way to get the plain keys out of KeyWallets.

This is the alternative (less breaking thanks) solution suggested by @klntsky in #859 

### Pre-review checklist

- [x] All code has been formatted using our config (`make format`)
- [x] The integration and unit tests have been run (`npm run test`) locally
- [x] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

**Review by klntsky — CHANGES_REQUESTED** [review]
Thanks!

We want to add getter functions and expose them as part of our `Contract` API. Say, from `Contract.Wallet`. 

`keyWalletPrivatePaymentKey`, `keyWalletPrivateStakeKey`.

Additionally, please add functions that allow to serialize both keys to strings in envelope format accepted by `cardano-cli`.  Should be the opposite of `privatePaymentKeyFromFile`, except that we want it pure, without writing to a file.

File: src/Wallet/KeyFile.purs [hunk 1]
```diff
@@ -45,3 +53,38 @@ privateStakeKeyFromFile filePath = do
...
+    , cborHex: _
+    }
+  <$> keyToCbor key
+
+keyToCbor :: PrivateKey -> Maybe String
+keyToCbor = ((rawBytesToHex >>> (magicPrefix <> _)) <$> _) <<<
+  bytesFromPrivateKey
+
+magicPrefix :: String
+magicPrefix = "5820"
```
Comment (Geometer1729) [inline src/Wallet/KeyFile.purs:90]: @klntsky Do you know what this comes from? It's needed to pass the tests but I have no idea why.

Comment (klntsky) [reply src/Wallet/KeyFile.purs:90]: It's from CBOR. 58 is a tag for array, 20 is byte length.

File: src/Wallet/KeyFile.purs [hunk 2]
```diff
@@ -3,10 +3,15 @@ module Wallet.KeyFile
   ( privatePaymentKeyFromFile
   , privateStakeKeyFromFile
   , keyFromFile
+  , privatePaymentKeyToFile
```
Comment (klntsky) [inline src/Wallet/KeyFile.purs:6]: Please re-export `formatStakeKey` and `formatPaymentKey` from `Contract.Wallet` and `privatePaymentKeyToFile` and `privateStakeKeyToFile` from `Contract.Wallet.KeyFile`.

**Review by klntsky — CHANGES_REQUESTED** [review]

File: src/Wallet/KeyFile.purs (hunk 1 above)
Comment (Geometer1729) [reply src/Wallet/KeyFile.purs:90]: Oh cool, thanks.

**Review by klntsky — APPROVED** [review]

**Pull request merged**